docs/multicast-guide.md

```diff
@@ -185,3 +186,7 @@ The configuration option `multicastInterfaces` is not supported with
 [Antrea FlexibleIPAM](antrea-ipam.md#antrea-flexible-ipam). When Antrea
 FlexibleIPAM is enabled, multicast packets are forwarded to the uplink interface
 only.
+
+### NetworkPolicyOnly mode
+
+Multicast is not supported with `networkPolicyOnly` mode.
```

Comment:
This should be at the very top of the document, right after the list of the 3 supported traffic patterns. We should have the sentence:

> Multicast is supported in `encap` and `noEncap` mode, but not in `networkPolicyOnly` mode